_posts/2016-10-03-sr2017-kickstart-dates-announced.md

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
Not sure if you can fake the date, but Jekyll does have the concept of drafts. If you put the post in the `_drafts` folder, it should get picked up by `rake dev`.